checker/src/main/java/org/checkerframework/checker/determinism/DeterminismVisitor.java

```diff
@@ -465,9 +465,15 @@ private void checkForDetConditional(ExpressionTree conditionalExpression) {
     }
 
     /**
-     * Reports an error if {@code @PolyDet("up")}, {@code @PolyDet("down")},
-     * {@code @PolyDet("use")}, or {@code @PolyDet("upDet")} is written on a formal parameter or a
-     * return type and none of the formal parameters or the receiver has the type {@code @PolyDet}.
+     * Reports an error if
+     *
+     * <ol>
+     *   <li>{@code @PolyDet("up")}, {@code @PolyDet("down")}, or {@code @PolyDet("upDet")} is
+     *       written on a formal parameter.
+     *   <li>{@code @PolyDet("up")}, {@code @PolyDet("down")}, {@code @PolyDet("upDet")}, or
+     *       {@code @PolyDet("use")} is written on a return type and none of the formal parameters
+     *       or the receiver has the type {@code @PolyDet}.
+     * </ol>
      */
```

Comment:
lThe local variable named `polyAnnotations` does not contain a set of poly annotations, but rather all annotations on the receiver and parameter.